plantio/urls.py

```diff
@@ -1,11 +1,14 @@
 from django.conf.urls import url
 from django.urls import include
 
-from plantio.views.plantio import PlantioAPIView
+from plantio.views.plantio import PlantioAPIView, PlantioAssociacaoAPIView
 
 
 urlpatterns = [
     url(r'^plantio/', include([
         url(r'^$', PlantioAPIView.as_view(), name='plantio-create'),
+        url(
+            r'^(?P<idPlantio>[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12})/$',
```

Comment:
Não ta muito expressiva
```suggestion
            r'^<str:uuid>/aplicacao/agrotoxico/$',
```

Reply:
acho que da pra colocar esse regex de uuid la no core pra ser reutilizado em outras rotas